Kernel/Library/StdLib.cpp

```diff
@@ -12,6 +12,25 @@
 #include <Kernel/Library/StdLib.h>
 #include <Kernel/Memory/MemoryManager.h>
 
+ErrorOr<void> try_copy_string_from_user(Userspace<char const*> user_str, char* destination_ptr, size_t user_str_size)
```

Comment:
Why not have this method return one of your fancy fixed size buffers, instead of taking a `char*` + size? This feels quite C-like, taking a char* as the destination rather than using our nice types, or even a writeable Span type.